src/Microsoft.ML.AutoML/Utils/BestResultUtil.cs

```diff
@@ -4,11 +4,36 @@
 
 using System.Collections.Generic;
 using System.Linq;
+using Microsoft.ML.Data;
```

Comment:
amazing, fast work, man. for this large change that was requested last-minute of you. thank you a lot